aequilibrae/paths/linear_approximation.py

```diff
@@ -114,6 +114,9 @@ def __init__(self, assig_spec, algorithm, project=None) -> None:
 
         # Instantiates the arrays that we will use over and over
         self.capacity = assig_spec.capacity
+        if assig_spec.apply_pt_preload:
```

Comment:
Nope...  Assignment is a load, not a deduction in capacity.

There is the obvious issue that pre-load could be larger than a link capacity (unlikely, but possible)

And this shifts the volume delay function, which is highly non-linear